.pre-commit-config.yaml

```diff
@@ -19,4 +19,5 @@ repos:
     - id: mypy
       # Don't run isolated but use the activated venv. Not perfect but better than re-declaring all deps here
       language: system
-      args: ['--strict', '--exclude', 'zavod/zavod/tests']
+      exclude: '^datasets|^zavod/zavod/tests'
+      args: ['--strict']
```

Comment:
@leonhandreke it looks like exclude doesn't work when the file is explicitly specified, e.g. when it's staged for commit but pre-commit also has an exclude option per-hook. Does this look ok to you?

```
mypy --strict --exclude zavod/zavod/tests --exclude datasets datasets/_global/afdb_sanctions/crawler.py
```

Reply:
If it works, looks good to me! Another way I just found is https://stackoverflow.com/questions/71657355/run-mypy-from-pre-commit-for-different-directories